jenkins/pipelines/post-build-status-update.jenkinsfile

```diff
@@ -29,6 +29,14 @@ pipeline {
 }
 
 def sendBuildStatus(repo, identifier, status, url, commit, ref) {
+
+    // Fail quietly if upstream job shouldn't be posting to GitHub.
+    if (!PR.isInteger() || POST_STATUS_TO_PR != 'true') {
```

Comment:
- Should `PR.isInteger()` change to checking if `ref` includes `refs/pull`? We never explicitly send `PR` to `post-build-status-update`

- `POST_STATUS_TO_PR` will need to be a new string parameter and variable passed to this function

Reply:
My bad on the first.

For the second, are parameters not passed down as environment variables in pipelines? If not that's a pain.

Reply:
So yes, the parameters are passed down as environment variables, but there are two annoying thing at play here -- 1) `sendBuildStatus` is a separate function, so we have to pass all parameters to it explicitly 2) `POST_STATUS_TO_PR` also needs to be defined as a parameter for the pipeline

Reply:
How about doing this (what I just pushed)?